umake/tools.py

```diff
@@ -337,6 +337,15 @@ def create_launcher(desktop_filename, content):
         gsettings.set_strv("favorites", launcher_list)
 
 
+def add_exec_link(exec_path, destination_name):
+        bin_folder = os.path.join(os.path.expanduser("~"), '.bin')
```

Comment:
so, expose that one in a property (to reuse the dir when removing). I agree, maybe change for the umake/bin subdirectory rather in local/share.